roles/uEsimulator/tasks/stop.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: delete {{ srsran.docker.container.ue_image }} container
+  community.docker.docker_container:
+    name: "rfsim5g-srsran-nr-ue"
+    state: absent
+  when: inventory_hostname in groups['srsran_nodes'][0]
+  become: true
+
+- name: remove /tmp/ue_zmq.conf
+  file:
+    path: "/tmp/ue_zmq.conf"
+    state: absent
+  when: inventory_hostname in groups['srsran_nodes'][0]
+  become: true
+
+  #TODO: Enable once we started using images from docker hub
+  #- name: remove {{ srsran.docker.container.ue_image }} image
+  #community.docker.docker_image:
+  #  name: "{{ srsran.docker.container.ue_image }}"
+  #  state: absent
+  #  force_absent: true
+  #when: inventory_hostname in groups['srsran_nodes'][0]
+  #become: true
```

Comment:
We already have the images in DockerHub (https://hub.docker.com/u/aetherproject), why keeping this TODO here?

Reply:
when this review raised we dont have images in DockerHub so i added this TODO. Now i enabled it.